src/spaced/spaced/game/hud.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <spaced/services/styles.hpp>
+#include <spaced/ui/outline_quad.hpp>
+#include <spaced/ui/text.hpp>
+#include <spaced/ui/view.hpp>
+
+namespace spaced {
+class Hud : public ui::View {
+  public:
+	explicit Hud(Services const& services);
+
+	void set_score(std::int64_t score);
```

Comment:
what's the benefit of `std::int64_t` over, say `uint64_t`?

Reply:
Unsigned integers don't behave like numbers and should be avoided.

https://hamstergene.github.io/posts/2021-10-30-do-not-use-unsigned-for-nonnegativity/